spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
> not just because it isn't part of the spec but also because all properties can be derived from raw source

Not all of them, and, well, Literal currently has only one apart from locations, and any other node doesn't have `raw` in this meaning.